### DISCUSSION TITLE
Colima support

### DISCUSSION THREAD

**Discussion by smiklos** [werf/werf#6507] [General]
### Before proceeding

- [X] I didn't find a similar [issue](https://github.com/werf/werf/issues)

### Problem

Is werf suppose to work for Mac M1+ users who run Colima instead of Docker Desktop? 

I tried to build an image without pushing it and it failed because it couldn't find the buildkit cache on my machine. 
When running werf like this, the command succeeds. 

```
DOCKER_BUILDKIT=0 werf build test
```

I'm using the docker runtime on Colima but it looks like buildkit is not part of the installation.

**Comment by iapershin** [comment]
Hi @smiklos. Since legacy builder is deprecated we use BuildKit builder by default. If you still want to use legacy builder you should set environment variable `DOCKER_BUILDKIT=0`. Hope it helps.
